tests/Unit/Console/Command/ReviewerPromoteCommandTest.php

```diff
@@ -0,0 +1,157 @@
+
+        $inputDefinition = $command->getDefinition();
+
+        $this->assertTrue($inputDefinition->hasArgument('email'));
+
+        $argument = $inputDefinition->getArgument('email');
+
+        $this->assertSame('Email address of user to promote to reviewer', $argument->getDescription());
+        $this->assertTrue($argument->isRequired());
+        $this->assertNull($argument->getDefault());
+        $this->assertFalse($argument->isArray());
+    }
+
+    public function testExecuteFailsIfUserDoesNotExist()
+    {
+        $email= $this->getFaker()->email;
+
+        $accountManagement = $this->createAccountManagementMock();
+
+        $accountManagement
+            ->expects($this->once())
+            ->method('findByLogin')
+            ->with($this->identicalTo($email))
+            ->willThrowException(new Sentry\Users\UserNotFoundException());
+
+        $command = new ReviewerPromoteCommand($accountManagement);
+
+        $commandTester = new Console\Tester\CommandTester($command);
+
+        $commandTester->execute([
+            'email' => $email,
+        ]);
+
+        $this->assertSame(1, $commandTester->getStatusCode());
+
+        $sectionMessage = sprintf(
+            'Promoting account with email "%s" to "Reviewer"',
+            $email
+        );
+
+        $this->assertContains($sectionMessage, $commandTester->getDisplay());
+
+        $failureMessage = sprintf(
+            'Could not find account with email "%s".',
+            $email
+        );
+
+        $this->assertContains($failureMessage, $commandTester->getDisplay());
+    }
+
+    public function testExecuteSucceedsIfUserExists()
+    {
+        $email = $this->getFaker()->email;
+
+        $user = $this->createSentryUserMock();
+
+        $accountManagement = $this->createAccountManagementMock();
+
+        $accountManagement
+            ->expects($this->at(0))
+            ->method('findByLogin')
+            ->with($this->identicalTo($email))
+            ->willReturn($user);
+
+        $accountManagement
+            ->expects($this->at(1))
+            ->method('promoteTo')
+            ->with(
+                $this->identicalTo($email),
+                $this->identicalTo('Reviewer')
+            );
+
+        $command = new ReviewerPromoteCommand($accountManagement);
+
+        $commandTester = new Console\Tester\CommandTester($command);
+
+        $commandTester->execute([
+            'email' => $email,
+        ]);
+
+        $this->assertSame(0, $commandTester->getStatusCode());
+
+        $sectionMessage = sprintf(
+            'Promoting account with email "%s" to "Reviewer"',
+            $email
+        );
+
+        $this->assertContains($sectionMessage, $commandTester->getDisplay());
+
+        $successMessage = sprintf(
+            'Added account with email "%s" to the "Reviewer" group',
+            $email
+        );
+
+        $this->assertContains($successMessage, $commandTester->getDisplay());
```

Comment:
This assertion fails due to newlines in the output from `getDisplay`:

```
1) OpenCFP\Test\Unit\Console\Command\ReviewerPromoteCommandTest::testExecuteSucceedsIfUserExists
Failed asserting that '\n
OpenCFP\n
=======\n
\n
Promoting account with email "adeline.stoltenberg@hotmail.com" to "Reviewer"\n
----------------------------------------------------------------------------\n
\n
 [OK] Added account with email "adeline.stoltenberg@hotmail.com" to the         \n
      "Reviewer" group                                                          \n
\n
' contains "Added account with email "adeline.stoltenberg@hotmail.com" to the "Reviewer" group".
```

I looked for a "nicer" way to work around this, but couldn't find anything. I *do* like that the expectations are super clear, so I'd personally be more on the side of transforming the output from `getDisplay` to make it pass (remove newlines / extra spaces)

Reply:
There's a simple way: setting an environment variable `COLUMNS` to a value that is large enough (otherwise it will be detected, or default to `80`, see [`Symfony\Component\Console\Terminal::getWidth()`](https://github.com/symfony/console/blob/v3.3.12/Terminal.php#L24-L36).